api_v3/lib/KalturaDispatcher.php

```diff
@@ -209,7 +210,11 @@ protected function validateUser($objectClass, $objectId, $privilege = null)
 			
 		}
 
-		if (strtolower($dbObject->getPuserId()) != strtolower(kCurrentContext::$ks_uid)) 
+		if (strtolower($dbObject->getPuserId()) != strtolower(kCurrentContext::$ks_uid)
+			&&
+			(!$dbObject->isEntitledKuserEdit(kCurrentContext::getCurrentKsKuserId())
+				||
+				($condition && $condition == KalturaDocCommentParser::OWNER_ONLY_CONDITION)))
```

Comment:
I think you better place the constant in this class (KalturaDispatcher) and not in the KalturaDocCommentParser.
I also would call the last argument $options, first because we might have options that are not conditions in the future, secondly, because we may have more than one option comma separated.